src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,123 @@
+@use "./variables" as *;
+
+.container {
+    display: grid;
+    grid-template-columns: repeat(1, $card-width);
+    margin: auto;
+    gap: 46px 48px;
+    padding: 50px 40px;
+    justify-content: center;
+    align-items: center;
+}
+
+.card {
+    width: 200px;
+    padding: 32px 16px 16px;
+    display: flex;
+    flex-direction: column;
+    border: 1px solid $grey-color;
+    border-radius: 5px;
+}
+
+.card__image {
+    width: 160px;
+    height: 134px;
+    background-repeat: no-repeat;
+    background-size: contain;
+    align-self: center;
+}
+
+.card__nameProduct {
+    font-size: 12px;
+    font-weight: 500;
+    margin-top: 40px;
+    line-height: 18px;
+    color: $main-color;
+    letter-spacing: 0;
+}
+
+.card__codeProduct {
+    margin: 4px 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-color;
+}
+
+.card__stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 12px;
+}
+
+.card__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    text-align: center;
+    color: $main-color;
+}
+
+.card__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 24px 0 16px;
+}
+
+.price-name {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $secondary-color;
+    text-align: left;
+}
+
+.price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    color: $main-color;
+}
```

Comment:
use `card__price-name` and `card__price-value`

Reply:
price-name and price-value classes amended